src/global/Pattern.h

```diff
@@ -128,7 +130,8 @@ class CompactVectorOfStrings {
 
   // Forward iterator for a `CompactVectorOfStrings` that reads directly from
   // disk without buffering the whole `Vector`.
-  static cppcoro::generator<vector_type> diskIterator(string filename);
+  static ad_utility::InputRangeTypeErased<vector_type> diskIterator(
```

Comment:
I think this function has been removed, so also remove the declaration here. Are the added includes above important, or are they an IDE artifact?

Reply:
oh yes, this is my fault, I will remove this.

Reply:
I forgot to do it [here](https://github.com/ad-freiburg/qlever/pull/2166), let it stay here please to save another pr.
I removed it [here](https://github.com/ad-freiburg/qlever/pull/2151/commits/a2f3cfe7e54dd02c567cfcdc54def6c7d57615a2)